Automattic/wp-calypso
# Add additional props in the calypso_signup_step_start event for the with-theme flow

Add the theme, theme_type and style_variation props to the calypso_signup_step_start event for the with-theme flow. 

With these new props will get more insights about what themes, their type and which style variation the user selects when their register from the with-theme flow. 

## Proposed Changes

* Add new tracks props to the with-theme flow.

## Testing Instructions



* Use the calypso live link and open an incognito window
* Go to /themes and click on a theme
* Open DevTools and filter the network requests by "t.gif"
* Click on the "Pick this design" button and you'll be redirected to signup page
* Notice that a new `calypso_signup_step_start` event was triggered 
* Make sure that the event contains the `theme`, `theme_type` and `style_variation` props (if you selected a style variation)
* Repeat this step for each type of theme (free, premium, WooCommerce, paid and DotORG). 
* Examples of themes (TT3, Alonso, Astra, Scentina)

## Pre-merge Checklist
